soc/arm/nxp_lpc/lpc55xxx/Kconfig.defconfig.lpc55S36

```diff
@@ -0,0 +1,27 @@
+# NXP LPC55S36 platform configuration options
+
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_LPC55S36
+
+config SOC
+	default "lpc55S36"
+
```

Comment:
remove PINMUX_MCUX_LPC, GPIO_MCUX_LPC, and UART_MCUX_FLEXCOMM.  They aren't needed.

Reply:
Already removed it.